cpp-compile.Dockerfile

```diff
@@ -10,56 +10,59 @@ ENV OS_DEP "\
       ca-certificates \
```

Comment:
Let's also update the base image to 19.04

Reply:
Would keep it like that for the moment. Let's support the local configuration for a bit longer